test/e2e/basic_test.go

```diff
@@ -0,0 +1,181 @@
+// +build e2e
+
+package e2e
+
+import (
+	"testing"
+	"time"
+
+	"github.com/yahoo/athenz/clients/go/zms"
+	"github.com/yahoo/k8s-athenz-syncer/test/e2e/framework"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+// TestBasicRoleUpdate - adds a new role in Athenz and tests if syncer updates the corresponding AthenzDomains resource
+func TestBasicRoleUpdate(t *testing.T) {
+	f := framework.Global
+	// pre-check to see if the target athenzdomains resource exists before modifying
+	_, exists, err := f.CRClient.GetCRByName(f.RoleDomain)
+	if err != nil {
+		t.Error("Pre-Check Error while finding athenzdomain resource from store")
```

Comment:
Can we add the error string to all log messages in all files? This will make it easier to debug the actual error.